50.NewAPITest/main.cpp

```diff
@@ -357,10 +357,10 @@ Choose Graphics API:
 		assert(buffer);
 
 		core::smart_refctd_ptr<video::IGPUCommandBuffer> cb;
-		device->createCommandBuffers(cmdpool.get(), video::IGPUCommandBuffer::EL_PRIMARY, 1u, &cb);
+		device->createCommandBuffers(cmdpool[0].get(), video::IGPUCommandBuffer::EL_PRIMARY, 1u, &cb);
```

Comment:
defaultCommandPool

Reply:
either @Crisspl or @sadiuk :D

Reply:
i dont remember, but in fact I wasn't caring about such details when writing tests for newest API XD it doesn't (didn't?) matter for GL backend anyway

Reply:
> i dont remember, but in fact I wasn't caring about such details when writing tests for newest API XD it doesn't (didn't?) matter for GL backend anyway

ah yeah it was because GL will only ever expose one queue family which can do everything.